backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/lang/SOTrx.java

```diff
@@ -108,4 +113,30 @@ public static SOTrx ofNameNotNull(@NonNull final String soTrx)
 					.setParameter("Known values", values());
 		}
 	}
+
+	@NonNull
+	public static SOTrx ofYesNoStringNotNull(@NonNull final String soTrx)
```

Comment:
consider renaming it to `ofYesNoString`
IMHO we don't need the "NotNull" part of the name because there is no other similar method here that returns null....